doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,473 @@
+.. _ncs_release_notes_160:
+
+Changes in |NCS| v1.6.0
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status. See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Added production support for nRF5340 for Bluetooth LE in single protocol configuration and in multiprotocol operation with Thread for Bluetooth Peripheral role.
```

Comment:
@jdstapes  @shantha-14  please check

Reply:
@emidttun FYI.

Reply:
@thomasstenersen / @jostn should the second part of sentence be removed. We know the root cause now.

Reply:
No, keep as is. The issue is more likely to present itself in the multiprotocol configuration.